src/Geolocation/SettingsGeolocation.php

```diff
@@ -94,6 +101,26 @@ public function getSettingsGlobalData(): array
 							<p>With every release we update that database but you can also provide your own database by using our filters. You can find more details <a href='%2\$s' rel='noopener noreferrer' target='_blank'>here</a>.</p>
 						", 'eightshift-forms'), 'https://www.maxmind.com', 'https://eightshift.com/forms/features/geolocation'),
 					],
+					[
+						'component' => 'divider',
+						'dividerExtraVSpacing' => true,
+					],
+					[
+						'component' => 'checkboxes',
+						'checkboxesFieldLabel' => '',
+						'checkboxesName' => UtilsSettingsHelper::getSettingName(self::SETTINGS_GEOLOCATION_COOKIELESS_USE_KEY),
+						'checkboxesFieldHelp' => \__("By default we store cookie for faster geodetection. Enabling this feature forms will never store geolocation cookie but keep in mind that this may increase detection time.", 'eightshift-forms'),
```

Comment:
```suggestion
						'checkboxesFieldHelp' => \__("By default we store a cookie for faster geo-location detection. By enabling this feature, forms will not store the geo-location cookie but keep in mind that this may increase geo-location detection time.", 'eightshift-forms'),
```